src/ui.rs

```diff
@@ -198,11 +199,18 @@ pub fn ui<B: Backend>(f: &mut Frame<B>, app_state: &mut AppState) {
                                 ),
                             ])
                         } else {
-                            Spans::from(vec![Span::raw(format!(
-                                "{}. {}",
-                                i + 1,
-                                choice.content.to_string()
-                            ))])
+                            Spans::from(vec![Span::styled(
+                                format!("{}. {}", i + 1, choice.content.to_string()),
+                                if let Some(value) = card.correct_answer {
+                                    if value {
+                                        Style::default().fg(Color::Green)
+                                    } else {
+                                        Style::default().fg(Color::Red)
```

Comment:
from our call, @BrookJeynes was saying we should create a  helper function to manage returned styles

something like this
```rust
let styled_text = |text: &str, colour: Color| {
  // return Span::Styled
}
```

Reply:
Resolved in https://github.com/BrookJeynes/quiz-rs/pull/4/commits/19cd7af19918eb404a73241e29f62765a5d0bd89